SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -347,6 +351,16 @@ Worker_RequestId USpatialWorkerConnection::SendDeleteEntityRequest(Worker_Entity
 	return NextRequestId++;
 }
 
+void USpatialWorkerConnection::SendAddComponent(Worker_EntityId EntityId, const Worker_ComponentData* ComponentData)
```

Comment:
Slightly tangential and the worker wrapper will end up replacing this kind of - this probably shouldn't be const as it is taking ownership. Even if the value doesn't change I don't think we should pass by pointer to const if ownership is transferred.

Reply:
In this case it has to take ownership since it'll be sent later.

Reply:
That is what I mean. As it takes ownership it's not const.

Reply:
Ah gotcha.